examples/lessons/07/app.js

```diff
@@ -3,205 +3,238 @@
-      <td>Y: <input type="text" id="lightDirectionY" value="-0.25" />
-      <td>Z: <input type="text" id="lightDirectionZ" value="-1.0" />
-      </tr>
-      <tr>
-      <td><b>Colour:</b>
-      <td>R: <input type="text" id="directionalR" value="0.8" />
-      <td>G: <input type="text" id="directionalG" value="0.8" />
-      <td>B: <input type="text" id="directionalB" value="0.8" />
-      </tr>
-      </table>
-
-      <h2>Ambient light:</h2>
-      <table style="border: 0; padding: 10px;">
-      <tr>
-      <td><b>Colour:</b>
-      <td>R: <input type="text" id="ambientR" value="0.2" />
-      <td>G: <input type="text" id="ambientG" value="0.2" />
-      <td>B: <input type="text" id="ambientB" value="0.2" />
-      </tr>
-      </table>
-
-      <a href="http://learningwebgl.com/blog/?p=684">&lt;&lt; Back to Lesson 7</a>
-    </div>
-  `;
-}
 
-function getControls() {
-  // Lighting form elements variables
-  var $id = function(d) {
-    return document.getElementById(d);
-  };
-
-  // Get lighting form elements
-  var lighting = $id('lighting');
-  var ambient = {
-    r: $id('ambientR'),
-    g: $id('ambientG'),
-    b: $id('ambientB')
-  };
-  var direction = {
-    x: $id('lightDirectionX'),
-    y: $id('lightDirectionY'),
-    z: $id('lightDirectionZ'),
-
-    r: $id('directionalR'),
-    g: $id('directionalG'),
-    b: $id('directionalB')
-  };
-
-  // Update scene config with light info
-  var lightConfig = scene.config.lights;
-  lightConfig.enable = lighting.checked;
-  lightConfig.ambient = {
-    r: Number(ambient.r.value),
-    g: Number(ambient.g.value),
-    b: Number(ambient.b.value)
-  };
-  lightConfig.directional.direction = {
-    x: Number(direction.x.value),
-    y: Number(direction.y.value),
-    z: Number(direction.z.value)
-  };
-  lightConfig.directional.color = {
-    r: Number(direction.r.value),
-    g: Number(direction.g.value),
-    b: Number(direction.b.value)
-  };
-}
-
-function addControls() {
-  const controlPanel = document.querySelector('.control-panel');
+function addControls({controlPanel} = {}) {
+  /* global document */
+  controlPanel = controlPanel || document.querySelector('.control-panel');
   if (controlPanel) {
     controlPanel.innerHTML = `
   <p>
-  A <code>cubemapped</code> prism within a larger cubemapped cube
+    <a href="http://learningwebgl.com/blog/?p=684" target="_blank">
+      Basic directional and ambient lighting
+    </a>
   <p>
-  Uses a luma.gl <code>TextureCube</code> and
-  the GLSL <code>reflect</code> and <code>refract</code> builtin functions
-  to calculate reflection and refraction directions from the prism normals
-  </p>
-  reflection
-  <input class="valign" id="reflection"
-    type="range" min="0.0" max="1.0" value="1.0" step="0.01">
-  <br>
-  refraction
-  <input class="valign" id="refraction"
-    type="range" min="0.0" max="1.0" value="1.0" step="0.01">
-  <br>
+    The classic WebGL Lessons in luma.gl
     `;
   }
 }
+
```

Comment:
Is this piece of code kept for adding functionality in the future? If yes, we should add a comment indicating that start with 'TODO' . Otherwise, we should remove it.